examples/Responsive.js

```diff
@@ -10,6 +21,7 @@ export default class Responsive extends Component {
       slidesToShow: 4,
       slidesToScroll: 4,
       initialSlide: 0,
+      unslick: UnSlick,
```

Comment:
It doesn't. 

example:
- I have N number of slides on every given page load. randomly from 2 to 6. 
- slides are not variable width, but they have a min-width and max-width which is declared by responsive settings. 
- On a wide screen the carousel settings `slidesToShow: 6` down to small `slidesToShow: 2`

the problem is: If `slide.count < settings.slidesToShow` then the carousel looks pretty silly: https://jsfiddle.net/20bumb4g/1388/

If i'm on a wide screen and 2 are returned from the server, the best way i've found to handle is converting to `UnSlick` and managing widths inside my Unslick component.  

I've built it into react-slick because it already contains all the necessary responsive utilities and breakpoint logic. Seems really silly to duplicate that inside & outside my component.